packages/lit-auth-client/src/lib/providers/EthWalletProvider.ts

```diff
@@ -92,7 +92,7 @@ export default class EthWalletProvider extends BaseProvider {
     } else {
       authSig = await checkAndSignAuthMessage({
         chain,
-        nonce: this.litNodeClient.latestBlockhash || generateNonce(),
+        nonce: this.litNodeClient.latestBlockhash!,
```

Comment:
Same here, we should check that `latestBlockhash` is defined before  attempting to sign the siwe